website/docs/markdown-features.mdx

```diff
@@ -568,6 +568,22 @@ module.exports = {
 };
 ```
 
+If you added a language under `prism.additionalLanguages` and used it on a code-block but that code-block is not getting highlighted, the reason might be missing styling support for tokens of that language in whatever syntax highlighting theme your Docusaurus config is using. You could manually add CSS, or you could import additional prism theme that will provide missing style for such language:
+```js {8} title="docusaurus.config.js"
+module.exports = {
+  // ...
+  presets: [
+    // ...
+    theme: {
+      customCss: [
+        require.resolve('./src/css/custom.css'),
+        'prismjs/themes/prism-dark.css'  // Additional prism theme.
```

Comment:
Thanks for the PR

Honestly I'm not even sure it's a good solution that we should recommend, as this newly loaded CSS can conflict with the ones we load by default, so it should rather be seen more as a workaround that a decent solution.

Unfortunately, I don't know what a good solution is exactly. I just know that some classes were missing, and this is a hacky way to possibly add the missing classes.

The proper solution is maybe related to these 2 existing attributes imho:

![image](https://user-images.githubusercontent.com/749374/95559765-9037c900-0a18-11eb-800d-28c9cff2d6c6.png)